test/python/test_build.py

```diff
@@ -94,3 +65,71 @@ def test_files_with_bad_symbols(cartridge_cmd, project_without_dependencies, tmp
     ]
     process = subprocess.run(cmd, cwd=project.path)
     assert process.returncode == 0, 'Building project failed'
+
+
+def test_app_without_rockspec(cartridge_cmd, project_without_dependencies):
```

Comment:
Could you push this tests to upstream?

Reply:
No, since now application build works without rockspec and hooks can be non-executable. It's an incompatible changes.